doc/articles/contributing/guidelines/testing.md

```diff
@@ -0,0 +1,4 @@
+# Guidelines for authoring tests
```

Comment:
🎗 need to add to the TOC

Reply:
Merge or pull content from https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md

Reply:
Let's:
- Burn the content of this file.
- Rename working-with-the-samples-apps.md to `doc/articles/contributing/guidelines/testing.md`
- Move all examples for code style into a separate document that contains all conventions -  https://github.com/unoplatform/uno/pull/1212/files